src/styles/_one-column.scss

```diff
@@ -0,0 +1,39 @@
+.one-column {
+    background-color: #1a3cfa;
+    height: auto;
+    color: white;
+
+    .container {
+
+        .content {
+            margin-top: 80px;
+            padding-top: 80px;
+            display: flex;
+            justify-content: center;
+
+            .column {
+                width: 60%;
+
+                .column-title {
+                    margin-top: 20px;
+                }
+
+                .column-btn {
+                    margin-top: 40px;
+                    margin-bottom: 80px;
+                }
+
+                .row {
+                    display: flex;
```

Comment:
You absolutely dont need to use flex everywhere. In this case you have just 1 item in each row. But strong side of flex - is a great logic of aligning multiple objects inside flex container. If you have just 1 object in each row - just user margin: 0 autol;